docs/reference/modules/threadpool.asciidoc

```diff
@@ -176,15 +178,14 @@ processors: 2
 There are a few use-cases for explicitly overriding the `processors`
 setting:
 
-. If you are running multiple instances of Elasticsearch on the same
-host but want Elasticsearch to size its thread pools as if it only has a
-fraction of the CPU, you should override the `processors` setting to the
-desired fraction (e.g., if you're running two instances of Elasticsearch
-on a 16-core machine, set `processors` to 8). Note that this is an
-expert-level use-case and there's a lot more involved than just setting
-the `processors` setting as there are other considerations like changing
-the number of garbage collector threads, pinning processes to cores,
-etc.
+. If you are running multiple instances of {es} on the same host but want {es} 
+to size its thread pools as if it only has a fraction of the CPU, you should 
+override the `processors` setting to the desired fraction (for example, if 
+you're running two instances of {es} on a 16-core machine, set `processors` to 
+8). Note that this is an expert-level use case and there's a lot more involved 
```

Comment:
We can probably just remove the parentheses from `(for example...` and treat it as a regular sentence.